index.html

```diff
@@ -0,0 +1,336 @@
+            </div>
+          </li>
+          <li class="speakers-list-items show-mobile hide">
+            <img
+              src="assets/images/speakers/Ohemaa.jpg"
+              width="178"
+              height="177"
+              class="speaker"
+            />
+            <div class="speaker-details">
+              <h3 class="name">Apst. Agartha. Owusu</h3>
+              <h4 class="position">
+                Global Chief Sheperd of Glorious Vineyard of Christ
+              </h4>
+              <p class="biography">
+                Aps. Agartha Owusu is the senior advisor of of the ministry. She
+                is in charge of giving counselling to the members to the
+                ministry.
+              </p>
+            </div>
+          </li>
+          <li class="speakers-list-items show-mobile hide">
+            <img
+              src="assets/images/speakers/dEBBY.jpg"
+              width="178"
+              height="177"
+              class="speaker"
+            />
+            <div class="speaker-details">
+              <h3 class="name">Min. Deborah Anaba</h3>
+              <h4 class="position">
+                Resident Pastor of Glorious Vineyard of Christ
+              </h4>
+              <p class="biography">
+                Min. Deborah is the Senior Watchman of the Sheep and Head
+                Minister in the ministryand oversee the activities of all other
+                Executives of the ministry.
+              </p>
+            </div>
+          </li>
+          <li class="speakers-list-items show-mobile hide">
+            <img
+              src="assets/images/speakers/Princess.jpg"
+              width="178"
+              height="177"
+              class="speaker"
+            />
+            <div class="speaker-details">
+              <h3 class="name">Min. Harriet Opoku</h3>
+              <h4 class="position">
+                Music Directress of Glorious Vineyard of Christ
+              </h4>
+              <p class="biography">
+                Min. Harriet oversee all the activites of the Music Ministry
+                including Choreography, Drama and Instrumentation and serves as
+                the Third Head Executives.
+              </p>
+            </div>
+          </li>
+          <li class="speakers-list-items show-mobile hide">
+            <img
+              src="assets/images/speakers/Desy.jpg"
+              width="178"
+              height="177"
+              class="speaker"
+            />
+            <div class="speaker-details">
+              <h3 class="name">Min. Desmond Opoku</h3>
+              <h4 class="position">
+                Assit. Evangelism Director of Glorious Vineyard of Christ
+              </h4>
+              <p class="biography">
+                Min. Desmond assist the Evangelism Director in the church and
+                also serve as the Head of the Children's Ministery.
+              </p>
+            </div>
+          </li> --> 
```

Comment:
- [ ] Please ensure that you remove the commented unused code in your code base, Commented code can clutter your repository and make it harder for others (and even yourself) to read and understand the active codebase. It can create confusion and increase the cognitive load when navigating the project.